critiquebrainz/db/users_test.py

```diff
@@ -0,0 +1,18 @@
+from sqlalchemy.exc import DatabaseError
+from critiquebrainz.data.testing import DataTestCase
+from critiquebrainz.data.model.user import User
+from critiquebrainz.db import users
+
+
+class UserTestCase(DataTestCase):
+    def setUp(self):
+        super(UserTestCase, self).setUp()
+        self.user1 = User.get_or_create("test", musicbrainz_id='0')
+        self.user2 = User.get_or_create("test1", musicbrainz_id='1')
+        self.admins = [self.user1.display_name, self.user2.display_name]
```

Comment:
Admins list probably shouldn't be in the `setUp` method since it's not going to be relevant to all function that will be in the `critiquebrainz.db.users` module. It just happens to be the case now. So I'd move that into the method that tests admin-related functions.

User definitions can be left there.